api/resources/applicants.py

```diff
@@ -35,6 +59,55 @@ def get(self, *args, **kwargs):
             'data': results
         }, 200
 
+    def post(self, *args, **kwargs):
+        applicant, errors = self._create_applicant(json.loads(request.data))
+
+        if applicant is not None:
+            applicant_payload = _applicant_payload(applicant)
+            applicant_payload['success'] = True
+            return {
+                'success': True,
+                'data': applicant_payload
+            }, 201
+        else:
+            return {
+                'success': False,
+                'error': 400,
+                'errors': errors
+            }, 400
+
+    def _create_applicant(self, data):
+        # helper method to create applicant and add to DB
+        proceed = True
+        errors = []
+
+        proceed, email, errors = _validate_field(
+            data, 'email', proceed, errors)
+
+        # skills and values arrays cannot be blank
+        proceed, skills, errors = _validate_field(
+            data, 'skills', proceed, errors)
+        proceed, values, errors = _validate_field(
+            data, 'values', proceed, errors)
+
```

Comment:
we should make sure FE knows to match this error handling on their end as well
@GreysonElkins @ErinUntermeyer @amykarnaze